docs.bruno/PingRoute.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: PingRoute
+  type: http
+  seq: 1
+}
+
+get {
+  url: http://localhost:3000/ping
```

Comment:
Consider adding error handling or a fallback mechanism in case the ping route fails or is unavailable. [important]